### PR TITLE
205 add form styles

### LINKED ISSUE #205: Add form styles to elements.erb

### PR DESCRIPTION
**Pull request by roidriscoll**
Closes #205

### CODE REVIEW

**Comment by stephenlloyd** [comment]
:+1:

File: source/sass/_rows.scss [hunk 1]
```diff
@@ -108,13 +116,13 @@
...
         @include border-left-radius($button-border-radius);
-        padding-left: modular-scale(0);
+        padding-left: $base-line-height;
         width: auto;
         margin: 0;
       }
       input[type="submit"] {
         @include border-left-radius(0);
-        padding: $form-input-padding modular-scale(0);
+        /*padding: $form-input-padding modular-scale(0);*/
```
Comment (pitchinvasion) [inline source/sass/_rows.scss]: comment

File: source/sass/_variables.scss [hunk 2]
```diff
@@ -35,6 +35,8 @@ $navy: #062533;
 $sea-blue: #00333F;
 $off-white: #EFF7F2;
 $red: #AA1925;
+$success-green: #387358;
```
Comment (pitchinvasion) [inline source/sass/_variables.scss]: I would create two variables `$green` and `$red` - and then set these to two variables `$success-color` and `$error-color`

**Comment by spike01** [comment]
Yah mon

**Comment by roidriscoll** [comment]
Looks fine with the intended date format:

screen shot 2015-07-14 at 13 34 40

**Pull request merged**